src/autoscaler/pruner/config/config_test.go

```diff
@@ -380,8 +372,8 @@ lock:
 				conf.Lock.ConsulClusterConfig = ""
 			})
 
-			It("should error", func() {
-				Expect(err).To(MatchError(MatchRegexp("Configuration error: Consul Cluster Config is empty")))
+			It("should validate successfully", func() {
+				Expect(err).NotTo(HaveOccurred())
```

Comment:
My point was that we don't need a test for optional parameters. And partial test is passing when consul cluster not specified, so it already validates what we are doing through this test case.